test.js

```diff
@@ -1,57 +1,56 @@
+import {Writable as WritableStream} from 'stream';
 import test from 'ava';
-import hookStd from 'hook-std';
 import Ora from './';
 
 const spinnerChar = process.platform === 'win32' ? '-' : '⠋';
 
-function readOutput(spinner, callback) {
-	let out = '';
-
-	const unhook = hookStd.stderr({silent: true}, output => {
-		out += output;
+let stream;
+let output;
```

Comment:
> Actually, tests are passing for Node.js v4 and v6 on Travis CI

Yeah, but they're fragile to race conditions.

-

Happy to answer any questions :)

AVA support channel: https://gitter.im/avajs/ava